fetch.bs

```diff
@@ -210,10 +210,36 @@ lt="authentication entry">authentication entries</a> (for HTTP authentication).
  <dt><dfn for="fetch params">cross-origin isolated capability</dfn> (default false)
  <dd>A boolean.
 
+ <dt><dfn for="fetch params">controller</dfn> (default a new <a for=/>fetch controller</a>)
+ <dd>A <a for=/>fetch controller</a>.
+
  <dt><dfn for="fetch params">timing info</dfn>
  <dd>A <a for=/>fetch timing info</a>.
 </dl>
 
+<p>A <dfn export>fetch controller</dfn> is a <a for=/>struct</a> used to enable callers of
+<a for=/>fetch</a> to perform certain operations on it after it has started. It has the following
+<a for=struct>items</a>:
+
+<dl>
+ <dt><dfn for="fetch controller">state</dfn> (default "<code>ongoing</code>")
+ <dd>"<code>ongoing</code>", "<code>terminated</code>", or "<code>aborted</code>"
```

Comment:
Should there be a state that represents a fetch that can no longer be aborted or terminated? I guess that would be "complete".

It seems weird that a fully completed fetch would have a state of "ongoing".

Reply:
Yes! I will add it when "Finalize and report timing" would move to the fetch controller, otherwise no one would set that value :)